src/libraries/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -3,6 +3,7 @@
     <RootNamespace>System.Diagnostics.Tests</RootNamespace>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
     <TestRuntime>true</TestRuntime>
+    <!-- some tests need types like System.Diagnostics.DebugProvider which are only exposed from System.Private.CoreLib -->
```

Comment:
```suggestion
    <!-- Some tests need types like System.Diagnostics.DebugProvider which are only exposed in the implementation assembly of System.Private.CoreLib -->
```

Reply:
that's not correct, it is exposed from the reference assembly of System.Private.CoreLib, just not System.Runtime.